paper/paper.md

```diff
@@ -1,42 +1,45 @@
 ---
```

Comment:
Additionally in the `paper.md` but not present in these changes:

- In table 1, `hydro electric` should be one word:  `hydroelectric`
- In the statement of need, the following reference is formatted and placed incorrectly - it should be rendered as:
>  In a study by Rice et. al (2013) which looked at de facto wastewater reuse across the US, it was found that there had been an increase in wastewater concentrations in drinking water treatment plants from a 1980 EPA report, especially at low flow conditions.